### PR TITLE
Add instructions on editing and viewing HTML

### PR DESCRIPTION
**Pull request by alope107**
Optional change, definitely feel free to reject this change if you disagree!

I figured since this activity asks students to write HTML, it's reasonable to suggest how to do that in VS Code and how to view their rendered file.

However, I could buy the argument that this activity is meant to get the students to focus on the structure of the HTML and not how it's presented, so it would be distracting to have them look at the rendered file.

Up to you to decide!

### CODE REVIEW

File: structuring-content/problem-set-structuring-content.md [hunk 1]
```diff
@@ -389,11 +389,21 @@ Do this work in a new `index.html` file on your computer, then reflect and answe
 
 Do not spend more than three hours on this. At three hours, consider it done! There is no submission for this. Be prepared to share your thought process with others.
 
+### !callout-info
+
+## Editing and Viewing HTML Files
+
+VS Code is a great editor for HTML files! It provides syntax hilighting and tag matching which makes writing your markup easier. 
+
+When you want to see how your HTML file is rendered, paste the full path to your file in the navigation bar of your browser. You can get the full path to a file in VS Code by holding control and clicking the name of the file, then selecting 'Copy Path'.
```
Comment (CheezItMan) [inline structuring-content/problem-set-structuring-content.md:398]: You may need to explain how to get the full path to a file here.  Most students are still iffy on filesystem paths in Mac/Unix systems.  

TBF most don't get it in Windows either.

**Review by CheezItMan — CHANGES_REQUESTED** [review]

File: structuring-content/problem-set-structuring-content.md (hunk 1 above)
Comment (alope107) [reply structuring-content/problem-set-structuring-content.md:398]: I added the note about how to get the path in VS Code. Do you think I should also add instructions about how to do it using `pwd` instead?

Comment (CheezItMan) [reply structuring-content/problem-set-structuring-content.md:398]: Knowing `pwd` is useful, but not essential.

Comment (alope107) [reply structuring-content/problem-set-structuring-content.md:398]: I agree. I think `pwd` might be a bit much to cram in here. So there's no change needed, right? The explanation of how to get the path from VS Code should be sufficient?

Comment (spitsfire) [reply structuring-content/problem-set-structuring-content.md:398]:
```suggestion
When you want to see how your HTML file is rendered, paste the full path to your file in the navigation bar of your browser. You can get the full path to a file in VS Code by holding `control` and clicking the name of the file (or right-clicking the file with a mouse), then selecting 'Copy Path'.
```

what do you think?

File: structuring-content/problem-set-structuring-content.md [hunk 2]
```diff
@@ -389,11 +389,21 @@ Do this work in a new `index.html` file on your computer, then reflect and answe
...
+
+### !end-callout
+
 ### !callout-success
 
 ## No Perfect Answers
 
-Translating content into HTML is creative work, with a lot of personal interpretation. Please keep in mind that there are no perfect answers!
+Translating content into HTML is creative work, with a lot of personal interpretation. Please keep in mind that there are no perfect answers! Your rendered HTML will look different from the sample resume and that's OK!
```
Comment (spitsfire) [inline structuring-content/problem-set-structuring-content.md:406]: 👍

**Review by spitsfire — APPROVED** [review]

**Review by CheezItMan — APPROVED** [review]
While I think Claire has good suggestions, this is pretty much good to go.

**Pull request merged**